evm_loader/program/src/config.rs

```diff
@@ -12,7 +12,7 @@ use crate::account_data::ACCOUNT_MAX_SIZE;
 cfg_if! {
     if #[cfg(feature = "mainnet")] {
 
-        const CHAIN_ID: u64 = 245022934;
+        const CHAIN_ID: u64 = 245_022_934;
```

Comment:
Why did you change the number format in this place and leave the old format in other branches of cfg_if ?
I think, the number format should be the same.

Reply:
We don't really need to change config.rs.
I reverted this.